nest/04.finest.md

```diff
@@ -0,0 +1,192 @@
+# 4 finest
+As explained throughout this getting-started, `finest` is the command targeted towards experienced users.
+
+## 4.1 Key concepts
+Before using `finest`, you have to understand how it works on a deeper level.
+`finest` relies on 4 key concepts:
+* The cache
+* Transactions
+* The dependency graph
+* Groups
+
+### 4.1.1 The cache
+[//]: # (TODO: add link to the section 3.2)
+The cache was explained in [the section 3.2]().
+
+### 4.1.2 Transactions
+A transaction is an *action of `finest` on the filesystem*.
+A transaction can be, for example, removing or upgrading a package.
+
+### 4.1.3 The dependency graph
+The dependency graph is a directed graph representing the dependencies of packages.
+In this graph, each node is either **a group of packages**, either **a single package**.
+Each node has a requirement that is fulfilled by another node.
```

Comment:
You often write short independent sentences.
That looks more like a list than an actual paragraph of text.
You could probably link those sentences.
(This is only one moment where I pin it)
(There are more)

Reply:
I tried to fix that.
In those two files.
Maybe later I will fix it in the other files.
If that same issue happened there too.
For now, this will do I guess.
I'll try not to do that for the getting-started of nbuild.